doc/modules/model_evaluation.rst

```diff
@@ -171,59 +196,61 @@ measuring a prediction error given ground truth and prediction:
-      ...     diff = np.abs(y_true - y_pred).max()
-      ...     return np.log1p(diff)
-      ...
-      >>> # score will negate the return value of my_custom_loss_func,
-      >>> # which will be np.log(2), 0.693, given the values for X
-      >>> # and y defined below.
-      >>> score = make_scorer(my_custom_loss_func, greater_is_better=False)
-      >>> X = [[1], [1]]
-      >>> y = [0, 1]
-      >>> from sklearn.dummy import DummyClassifier
-      >>> clf = DummyClassifier(strategy='most_frequent', random_state=0)
-      >>> clf = clf.fit(X, y)
-      >>> my_custom_loss_func(y, clf.predict(X))
-      0.69...
-      >>> score(clf, X, y)
-      -0.69...
+Custom scorer objects using `make_scorer`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The second use case is to build a completely custom scorer object
+from a simple python function using :func:`make_scorer`, which can
+take several parameters:
+
+* the python function you want to use (``my_custom_loss_func``
+  in the example below)
+
+* whether the python function returns a score (``greater_is_better=True``,
+  the default) or a loss (``greater_is_better=False``).  If a loss, the output
+  of the python function is negated by the scorer object, conforming to
+  the cross validation convention that scorers return higher values for better models.
+
+* for classification metrics only: whether the python function you provided requires
+  continuous decision certainties. If the scoring function only accepts probability
+  estimates (e.g. :func:`metrics.log_loss`) then one needs to set the parameter
+  `response_method`, thus in this case `response_method="predict_proba"`. Some scoring
+  function do not necessarily require probability estimates but rather non-thresholded
+  decision values (e.g. :func:`metrics.roc_auc_score`). In this case, one provides a
+  list such as `response_method=["decision_function", "predict_proba"]`. In this case,
+  the scorer will use the first available method, in the order given in the list,
+  to compute the scores.
+
+* any additional parameters, such as ``beta`` or ``labels`` in :func:`f1_score`.
+
+Here is an example of building custom scorers, and of using the
+``greater_is_better`` parameter::
+
+    >>> import numpy as np
+    >>> def my_custom_loss_func(y_true, y_pred):
+    ...     diff = np.abs(y_true - y_pred).max()
+    ...     return np.log1p(diff)
+    ...
+    >>> # score will negate the return value of my_custom_loss_func,
+    >>> # which will be np.log(2), 0.693, given the values for X
+    >>> # and y defined below.
+    >>> score = make_scorer(my_custom_loss_func, greater_is_better=False)
+    >>> X = [[1], [1]]
+    >>> y = [0, 1]
+    >>> from sklearn.dummy import DummyClassifier
+    >>> clf = DummyClassifier(strategy='most_frequent', random_state=0)
+    >>> clf = clf.fit(X, y)
+    >>> my_custom_loss_func(y, clf.predict(X))
+    0.69...
+    >>> score(clf, X, y)
+    -0.69...
 
 .. _diy_scoring:
```

Comment:
By doing a grep it seems that `diy_scoring` is not referenced elsewhere in the documentation. Maybe the preamble paragraph here was meant to avoid breaking the cross-reference, but can now be completely inside the dropdown, to simplify a bit.

Reply:
To clarify, are you suggesting that we move the "Custom scorer objects from scratch" section into a dropdown, and then we'd rename the above "Custom scorer objects using `make_scorer`" section to just be called "Custom scorer objects" ?